solidity/contracts/utils/BytesLib.sol

```diff
@@ -426,4 +426,29 @@ library BytesLib {
             result := keccak256(add(add(_bytes, 32), _start), _length)
         }
     }
+
+    function toBytes(address _address) internal pure returns (bytes memory) {
+        bytes memory result = new bytes(20);
+        for (uint256 i = 0; i < 20; i++) {
+            result[i] = bytes20(_address)[i];
+        }
+    }
+
+    function overwrite(
```

Comment:
For managed grants, not in the branch yet

Reply:
Let's then remove this function from this PR and add in the managed grants branch, it will be much easier to review and we'll avoid potential dead code if we don't eventually decide to use it.